config/mocks/wallet-options-v4.json

```diff
@@ -28,6 +28,24 @@
         "walletConnect": true,
         "withdrawalLocksFundsOnHold": true,
         "useNewPaymentProviders": true
+        "paymentProcessors": true
+      },
+      "hotWalletAddresses":{
+        "swap": {
+          "eth": "0xC05f893248D843192A989AEE6dC26d21426E78AF"
+        },
+        "exchange": {
+          "eth": "0x95eFeFA0A4564BfABfBB1Ee08A359C062EB0A94b"
+        },
+        "simplebuy": {
+          "eth": "0xFF202CBC6C794516164E45621143acDe4a3FEa24"
+        },
+        "lending": {
+          "eth": "0xBA0145434A0a53E58D10680254756F679c359463"
+        },
+        "rewards": {
+          "eth": "0x7d9d70c1A99FcC952d48b781947FAa0a3795083E"
+        }
```

Comment:
cc @schnogz should we also set up staging.js, development.js, and production.js values and apply in webpackBuild?

Reply:
i think this is covered via the applications PR that was created and merged